website/content/docs/v0.7/Local Platforms/virtualbox.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Virtualbox
```

Comment:
nit: The official title is `VirtualBox` (capital B). Looks like this will need to be updated in other places too.

Reply:
fixed!